__tests__/reducers/welcome.test.js

```diff
@@ -28,3 +29,11 @@ describe('TOGGLE_FADE', () => {
     expect(result).toMatchSnapshot()
   })
 })
+
```

Comment:
I added that function from the settings reducer tests since i'm using the same function in the welcome reducer to save the language. If is wrong let me know and i delete it